sites/all/themes/orwell/sass/component/_ting-object.scss

```diff
@@ -249,74 +249,74 @@ $material--teaser__margin-bottom: 1.2rem;
       }
     }
   }
-}
-
-.action-button {
-  @extend .btn;
-  float: left;
-  text-align: left;
-
-  @include grid-media($medium) {
-    max-width: 47%;
-  }
-}
-
-.content-wrapper--material {
-  padding: 0 2rem;
-}
-
-.detail {
-  width: 100%;
-  display: inline-block;
-  clear: left;
-  padding: 1.5rem 0;
-  border-top: 1px solid $color__border-gray;
-  margin: 0;
 
-  .article--search-result {
+  .action-button {
+    @extend .btn;
     float: left;
-    width: 100%;
-    margin: .5rem 0;
+    text-align: left;
 
     @include grid-media($medium) {
-      margin: 1.5rem 0 2rem;
+      max-width: 47%;
     }
   }
-}
 
-.group-material-details .field-group-format-wrapper {
-  font-family: $font-regular;
-  font-weight: normal;
+  .detail {
```

Comment:
The horse is not quite dead yet: Sooo, what a ".detail", why does it need a clear left? etc....

Reply:
Hell if I know? I just fixed the nesting...